src/sagemaker/session.py

```diff
@@ -3055,6 +3055,11 @@ def create_endpoint_config_from_existing(
         if request_data_capture_config_dict is not None:
             request["DataCaptureConfig"] = request_data_capture_config_dict
 
+        if existing_endpoint_config_desc.get("AsyncInferenceConfig") is not None:
```

Comment:
can you please change " existing_endpoint_config_desc.get("AsyncInferenceConfig") " to " existing_endpoint_config_desc.get("AsyncInferenceConfig", None) "

Reply:
Sure

Reply:
Oh I see it now I meant it on line 58